content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,27 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based exclusion filter
+
+In addition to the default sampling on the raw log (exclusion percentage set on `all logs`), you can choose to sample on a specific attribute holding an ID, like `UserID`, `SessionID`, `RequestID`.
```

Comment:
"raw log" ?
-> "such as User, Session or Request Identifiers" (actual information rather than example path name seems more relevant here)

Reply:
Thanks for the review.
Implementing changes.